.gitattributes

```diff
@@ -1,3 +1,7 @@
+# Git will always convert line endings to LF on checkout.
+# You should use this for files that must keep LF endings, even on Windows.
+* text eol=lf
```

Comment:
what about the auto part? https://github.com/cakephp/cakephp/blob/master/.gitattributes#L3